foursight_core/deploy.py

```diff
@@ -150,15 +150,20 @@ def build_config(cls, stage, identity=None, stack_name=None,
         print(''.join(['Writing: ', filename]))
         with open(filename, 'w') as config_file:
             config_file.write(json.dumps(cls.CONFIG_BASE))
+        print(''.join(['Done writing: ', filename]))
         # export poetry into requirements
+        print('Executing: poetry export -f requirements --without-hashes -o requirements.txt')
         subprocess.check_call(
             ['poetry', 'export', '-f', 'requirements.txt', '--without-hashes', '-o', 'requirements.txt'])
+        print('Done executing: poetry export -f requirements --without-hashes -o requirements.txt')
```

Comment:
It might be worth making a subroutine, which we could then at some point move to `dcicutils.command_utils`. It could look like this (untested):
```
def subprocess_call(command, **kwargs):
    command_str = ' '.join(command)
    print(f'Executing: {command_str}')
    result = subprocess.check_call(command, **kwargs)
    print(f'Done executing: {command_str}')
    return result
```
One reason I like this approach, besides just textual convenience, is that it's harder to get the print statements out of sync with the command. A hasty edit to the command might forget to update them, and the result could be really confusing.

That would allow someone to do (untested):
```suggestion
        subprocess_call([
            'poetry', 'export', '-f', 'requirements.txt', '--without-hashes', '-o', 'requirements.txt'
        ])
```